src/pages/SinglePracticePage/styled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const SinglePracticeWrapper = styled.section`
+  .single-practice-page__hero-practice {
+    margin-bottom: 120px;
+  }
+  .single-practice-page__link-box-container {
+    margin-bottom: 80px;
+  }
+  .single-practice-page__shortcut-link-container {
+    margin-bottom: 80px;
+  }
```

Comment:
Any reason why CSS classes are used instead of styled components?

Reply:
Yeah, but maybe not a good reason :) 

It was done to replace the <Space/> component which rendered an empty div with a bottom-margin set to 80px.

I haven't really gotten much experience with styled components, so I wasn't entirely sure how I would go about selecting the elements with need of a bottom margin, since the classNames for the components are generated. Is there a better way to do this with styled components?

I could of course just moved the margin-bottom into the styled-components themselves. I just wasn't sure if those components are going to be reused on other pages, which might lead to some spacing issues on those pages(?). This might just be med overthinking a non-issue though :D

Reply:
No worries, it's good of you to think about whyd and hows!

Generally speaking, we rarely used CSS classes together with styled components. The reason for this is because we only want to affect the style of the actual component being styled, _not its children_. What I'm trying to say here is that we rarely want parent components to know about the implementation of their child components and vice versa, as this creates a direct dependency that is more susceptible for breaking changes. For instance, if someone decides to replace SinglePageWrapper with a different wrapping in the future - without checking the implementation - then it suddenly also affects the margins of child elements. This is bad because the _interface_ and semantics of SinglePageWrapper does not tell the user anything about a margin feature for children.

As such, it makes sense to have a styled component that represents a block/wrapper with a margin a the bottom (and maybe it could take in the size of the margin as a prop?), given that there are more places that might want such a margin.

Alternatively, you can keep the CSS classes, but move them into the global stylesheet instead. In this case, there is no inter-component dependencies, as global styles are like utility styles that can be sprinkled throughout the app.